src/markdown/src/components/styled/MarkdownWrapper.js

```diff
@@ -0,0 +1,98 @@
+import styled from 'styled-components';
+
+const MarkdownWrapper = styled.div`
```

Comment:
Does this file really need to exist? It's only being used in one place, why not just inline the component there?

Reply:
sort plz.

Reply:
I am not going to sort this because some of the styles cascade into each other. It does not make sense to sort css classnames alphabetically.

This file is kept separately for visual clarity and separation of concerns. Since it's just styles, it's being treated as such.